src/mugs.js

```diff
@@ -295,7 +295,45 @@ define([
             rawControlXML: {
                 presence: 'optional',
                 lstring: 'Raw XML'
-            }
+            },
+            dataParent: {
+                lstring: 'Data Parent',
+                visibility: function(mug) {
+                    function recFunc(mug) {
+                        if (!mug) {
+                            return true;
+                        } else if (mug.__className === 'Repeat' ||
+                                   mug.__className === "Item") {
```

Comment:
Could this be changed to `mug.paretMug && !mug.parentMug.options.possibleDataParent`

Is this a correct statement?
A repeat node can have a data parent, but a repeat cannot itself be a data parent.

Hmm, is this code enforcing that?